ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsInitializeIngestion.scala

```diff
@@ -30,6 +30,7 @@ private[backend] trait StorageBackendTestsInitializeIngestion
       dtoPackageEntry(offset(3)),
       // 4: transaction with create node
       dtoCreate(offset(4), 1L, "#4"),
+      DbDto.CreateFilter(1L, someTemplateId.toString, someParty.toString),
```

Comment:
```suggestion
      dtoCreateFilter(1L, someParty),
```
we could add a helper like for all other DTOs

Reply:
sure why not